client/my-sites/site-settings/search.jsx

```diff
@@ -286,7 +286,7 @@ class Search extends Component {
 				</CompactCard>
 				{ hasSearchProduct && fields.instant_search_enabled && (
 					<CompactCard
-						href={ this.props.customizerUrl }
+						href={ `${ this.props.siteAdminUrl }admin.php?page=jetpack-search-configure` }
```

Comment:
I think we might need to do the Jetpack version compare here too?

I have a Jetpack test site (non-Atomic) that appears in my Calypso site list, but is still on Jetpack 10.1. I get an unauthorized error when I click this link from `/settings/performance`.

Reply:
Good point, I'll look into this.

Reply:
This has been addressed via the addition and usage of the new `getJetpackSearchCustomizeUrl` selector.